jsx/FilterableDataTable.js

```diff
@@ -1,10 +1,11 @@
-import React, {Component} from 'react';
+import React, {PureComponent} from 'react';
 import PropTypes from 'prop-types';
 
 import Panel from 'jsx/Panel';
 import {Tabs, TabPane} from 'jsx/Tabs';
 import DataTable from 'jsx/DataTable';
 import Filter from 'jsx/Filter';
+import LoadingBar from 'jsx/LoadingBar';
```

Comment:
not in **24.1**

_this is an important UI feature of the biobank_

include as  **CBIG override** and push to **main**